lidarSuit/windPropRetrieval.py

```diff
@@ -362,7 +362,7 @@ def correctWindComp(self, comp):
         """
 
         self.logger.info(
-            "replacing the gate_index coordinate by range: {0}".format(
+            "replacing the gate_index coordinate by range: {}".format(
```

Comment:
I think this one needs a manual correction.

Reply:
Good catch. Thanks @jdiasn